scripts/images/convert_qcow2_ova.py

```diff
@@ -84,7 +84,7 @@
 """
 
 template_rhel_bash = """#!/bin/bash
-if [ "{{ distribution }}" == "rhel"];then
+if [ "{{ distribution }}" == "rhel" ];then
```

Comment:
I insist to include these just after the bash interpreter to catch any failures!

```
set -o errexit
set -o nounset
set -o pipefail
set -o xtrace
```